datastore/storemanager.go

```diff
@@ -396,9 +398,13 @@ func (sm *StoreManager) flush(terminating bool) {
 func (sm *StoreManager) query(q config.MetricQuery) {
 	switch strings.ToLower(q.Method) {
 	case "delete":
+		mqdt := time.Now()
```

Comment:
If you always use the same variable name for the stored start time, you make it harder for someone to make a copy/paste error later.